ui/src/main/resources/PhenoTips/MendelianSearch.xml

```diff
@@ -483,13 +483,9 @@ div.mendelian-search-symptoms {
 .variant-categories {
```

Comment:
Do we really need to keep the background and the border? The UI is cleaner without them, IMO.